examples/dashboard_basic_crud.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+#
+# Simple example of dashboard creation, retrieval, updating, and deletion.
+#
+
+import os
+import sys
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), '..'))
+from sdcclient import SdMonitorClient
+import uuid
+
+#
+# Parse arguments
+#
+if len(sys.argv) != 2:
+    print('usage: %s <sysdig-token>' % sys.argv[0])
+    print('You can find your token at https://app.sysdigcloud.com/#/settings/user')
+    sys.exit(1)
+
+sdc_token = sys.argv[1]
+
+#
+# Instantiate the SDC client
+#
+sdclient = SdMonitorClient(sdc_token)
+
+#
+# Create Dashboard.
+#
+ok, res = sdclient.create_dashboard("Sample dashboard - " + uuid.uuid4().hex)
+
+#
+# Check for successful creation
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+dashboard = res['dashboard']
+
+#
+# Get Dashboard.
+#
+ok, res = sdclient.get_dashboard(dashboard['id'])
+
+#
+# Check for successful retrieval
+#
+if not ok:
+    print(res)
+    sys.exit(1)
+
+dashboard = res['dashboard']
+
+#
+# Update Dashboard.
+#
+dashboard['scopeExpressionList'] = [{
```

Comment:
As much as useful, this update seems to shift the focus to "how to set a valid dashboard scope". It may be a little distracting or misleading. I'd suggest a simpler example like
```
dashboard['name'] = "Another sample dashboard - " + uuid.uuid4().hex
```

However, this makes me think that it could be very useful to have a separate place where to document useful snippets like this one (panels and sharing options are other 2 potential good use cases for this document).

Hope this make sense :-)